Jetsnack/app/src/main/java/com/example/jetsnack/model/Snack.kt

```diff
@@ -41,183 +42,183 @@ val snacks = listOf(
         price = 299
     ),
     Snack(
-        id = 2L,
+        id = Random.nextLong(),
```

Comment:
Cupcake above still has a fixed ID. Do these IDs need to be random? Also, isn't it now possible (albeit very improbable) that two snacks end up with the same ID?

Reply:
Its for preview purposes to be able to query for a certain snack. yes it is somewhat possible, but this is fake mock data so I dont think its worth the hassle of making it more robust.